docs/interactive/launchbuttons.ipynb

```diff
@@ -19,7 +19,24 @@
     "  url                       : https://github.com/yourusername/yourbookrepo  # Online location of your book\n",
     "  path_to_book              : path/to/book  # Optional path to your book, relative to the repository root\n",
     "  branch                    : master  # Which branch of the repository should be used when creating links (optional)\n",
-    "```"
+    "```\n",
+    "The launch button will use [nbgitpuller](https://github.com/jupyterhub/nbgitpuller) to pull your\n",
+    "repository to the chosen Binder, Google Colab, or JupyterHub location.  The `repository:`\n",
+    "configuration line uses `url`, `path_to_book`, and `branch` to find your notebook files _both MyST\n",
+    "Markdown and *.ipynb_.  \n",
+    "\n",
+    "- The `url` should be a GitHub repository that includes all of your files used to build the\n",
+    "  Jupyter-book. Your repository should include:\n",
+    "    - `_build`\n",
```

Comment:
actually generally speaking we *don't* want people to put the `_build` folder in their source repository. There are usually two places (often corresponding to 2 branches) to consider:

- Your book's source files. These are the config, notebooks, etc files that make up your book
- Your book's *build assets*. These are created by `jupyter-book build`. 

For launch buttons, they need to point to the *source files*, not the assets.

Reply:
Thanks for clarifying. I had misinterpreted the instructions. 

In another section of the docs, to get the [repo set up](https://jupyterbook.org/start/publish.html) the suggested step is to 
`git add ./*` 

and [building the book](https://jupyterbook.org/start/build.html) results in the _build directory being included in the main repository folder so it gets swooped into the `add ./*`. 

Maybe, we could clarify the source-vs-build files in the publish and build docs.

Reply:
I removed `_build` from the list of files, but what would a minimal github repo include:
* `_config.py` _remove?_
* `conf.py` _remove?_
* `requirements.txt` _remove?_
* source files

Reply:
basically just `_config.yml`, `_toc.yml`, and source files are what is needed.

Reply:
Is the `requirements.txt` just for binder? 

Should I leave it in the list of files and add a note e.g.
"

* `requirements.txt` : packages for binder to build

"

Reply:
Yep that works - correct that requirements.txt is not strictly required, just for binder or general reproducibility